libs/sr/solver/src/parentWorker.ts

```diff
@@ -92,7 +92,30 @@ async function start({
     relicsBySlot[slot].map(convertRelicToStats)
   )
 
-  const chunkSize = Math.ceil(relicsBySlot.head.length / numWorkers)
+  // Split work on the largest slot to reduce variance in work between workers
+  const { largestSlot } = Object.entries(relicsBySlot).reduce(
+    ({ largestSlot, largestSize }, [currentSlot, relics]) =>
+      relics.length > largestSize
+        ? { largestSlot: currentSlot, largestSize: relics.length }
+        : { largestSlot, largestSize },
+    { largestSlot: 'head' as RelicSlotKey, largestSize: -1 }
+  )
+  // Split work
+  const chunkedRelicStatsBySlot: Record<RelicSlotKey, RelicStats[]>[] = range(
+    0,
+    numWorkers - 1
+  ).map((worker) =>
+    objKeyMap(allRelicSlotKeys, (slot) =>
+      slot === largestSlot
+        ? // For largest slot, only give ~1/numWorkers amount of relics
+          relicStatsBySlot[slot].filter(
```

Comment:
What happens in the case where there are more `numWorker` than relics in `largestSlot`? does worker work with an empty slot array?

Reply:
It works as expected. For the 24 permutation case, the largest slot was 3 and I had 8 workers. 8 workers still spun up but only 3 did actual work. Could optimize more by not spinning up those workers. But in this case, the work should be done super fast anyways